e2e/testcases/helm_sync_test.go

```diff
@@ -200,6 +202,18 @@ func TestHelmLatestVersion(t *testing.T) {
 		nt.T.Fatalf("failed to push helm chart: %v", err)
 	}
 
+	nt.T.Log("Update reconciler-manager-cm to override the helm-sync default poll period")
```

Comment:
Similar test-related alterations to things like polling periods are done [here](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/nomostest/config_sync.go#L142). Suggest doing it there for consistency and so that it is set for all tests by the test scaffolding

Reply:
Thanks for the tip! Will update

Reply:
So I discovered that the current way that polling periods are overridden in tests is actually not working, because it is updating the data in [reconciler-manager-cm](https://github.com/GoogleContainerTools/kpt-config-sync/blob/5ffda55c6c928d234206af6e9233451969151eff/e2e/nomostest/config_sync.go#L578), but the environment is mounted from [reconciler-manger](https://github.com/GoogleContainerTools/kpt-config-sync/blob/5ffda55c6c928d234206af6e9233451969151eff/manifests/templates/reconciler-manager.yaml#L57) (which AFAIK is a ConfigMap that is only used for e2e tests, so it makes sense to put the poll overrides there). I added some logging and saw that yup, during e2e tests, the reconciler and hydration polling periods are still currently set to their default, instead of the having override that they're supposed to have.

I included a fix to the `setPollingPeriod` function in `nomostest` to fix that. And updated this helm-sync test accordingly, to follow a similar model.

Reply:
Looks like using the sped up reconciler and hydration polling periods caused some presubmits to fail, I'm increasing them to the default to see if it fixes the presubmit

Reply:
It looks like increasing the reconciler/hydration polling periods back to their default fixed the test failures. So this PR is essentially leaving the hydration/reconciler poling delays as it was before (since the override wasn't working). It is just modifying the helm-sync-version polling period.

I think we can play around with modifying the hydration/reconciler polling periods as a followup, but I don't think it belongs in this PR since it is unrelated to the helm-sync changes.

Reply:
Good catch, sounds like it was bugged before. Thank you for fixing, agreed that you should just leave the other overrided values at their default and it can be investigated separately